src/main/java/io/vertx/core/http/impl/HttpUtils.java

```diff
@@ -380,6 +380,15 @@ static Http2Settings decodeSettings(String base64Settings) {
     return null;
   }
 
+  public static String encodeSettings(io.vertx.core.http.Http2Settings settings) {
```

Comment:
This is for test only right? Shouldn't it move to test dir then?

Reply:
I think it's fine because it's in impl